CHANGELOG

```diff
@@ -1,6 +1,7 @@
 UNRELEASED
     * Fix crash on html-mail entries with no URL
     * Add a new `subject-format` setting, customise the subject line
+    * Removed '$' interpolation in config file to allow URLs containing dollar signs. Interpolation was not fully supported, if at all, and the placeholder would not survive a save. Therefore, any prior use of interpolation will continue to work since the placeholder was not written to the config file.
```

Comment:
I think the “Therefore” here is technically wrong: people who wrote out r2e's config file by hand and only ever used `r2e run` probably could have been using interpolation successfully.

So it may be best to just remove the “Therefore” line and just let people check whether they had `$`'s in their config files themselves upon upgrading. WDYT?

Reply:
I tested the interpolation and it does work using ${option} or ${section:option}. Not sure how useful that would be to anyone, but it works, until a save overwrites it. This patch only affects those using the master branch because `r2e run` saved the config prior to #186 which was merged one month after the last release. But I updated the CHANGELOG message for anyone who might have used interpolation.

Reply:
Awesome, thanks!